### ISSUE TITLE
Plan and create the Software Architecture PDF

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#12]
## User story
1. As a Stakeholder
2. I want to have a visualized software architecture
3. So that I can understand how the product/software works

## Acceptance criteria
* Create a PDF containing the following information:
   * [ ] An overview diagram of runtime components (one page)
   * [ ] An overview diagram of code components (one page)
   * [ ] A summary of the underlying technology stack (one page)
   * [ ] A textual explanation of the diagrams and choices (one page)

* Add the document to the deliverables folder: '_sprint-02/software-architecture.pdf_'

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by jnsrnhld** [comment]
Just small draft for the runtime components : https://drive.google.com/file/d/1VoZqjwQT7gaYha3A7xuvP96DCODYMHQh/view?usp=sharing

![Image](https://user-images.githubusercontent.com/85998450/234572194-d5729501-af48-4209-8db7-e41274e60490.png)

Feel free to edit, everyone with the link should have access.

**Comment by ahaverkamp** [comment]
Added a rough overview of our potential backend code components on the second page:
https://app.diagrams.net/#G1VoZqjwQT7gaYha3A7xuvP96DCODYMHQh

![codeComponents](https://user-images.githubusercontent.com/61870854/235481883-82cc4486-0cd6-4580-839b-cad991dcdbc7.png)
